addons/website_slides/controllers/main.py

```diff
@@ -368,3 +377,104 @@ def slides_embed(self, slide_id, page="1", **kw):
         except AccessError: # TODO : please, make it clean one day, or find another secure way to detect
                             # if the slide can be embedded, and properly display the error message.
             return request.render('website_slides.embed_slide_forbidden', {})
+
+    # Profile
+    # ---------------------------------------------------
+    def _prepare_open_slide_user(self, user):
+        slides = request.env['slide.slide'].sudo().search([('user_ids', '=', user.id)])
```

Comment:
To check more in details: what are the required details that we display on a given user statistics ? Indeed as everything is sudo-ed you could grant information to people that should not see it.

If we display only "number of" -> prepare those integers and don't return sudo-ed records
If we display activity -> it should be browsed using the current user's access rights so that information related to private slides / channels is not displayed to external people. Otherwise this is information leak.